apps/settings/locales/settings.en-US.properties

```diff
@@ -546,31 +540,8 @@ hotspotSettings-header={{hotspotSettings}}
 ssid-name=SSID Network Name
 hotspot-open=open
 
-# LOCALIZATION NOTE
-#
-# [For localizers]
-# We can't render these strings correctly for RTL locales, so we have to put LRE (&#x202a;)
-# and PDF (&#x202c;) entities to these translations to force the correct direction.
-#
-# RTL locales should have a translation similar to this:
-# hotspot-wpa-psk.innerHTML= &#x202a;WPA (TKIP)&#x202c;
-# hotspot-wpa2-psk.innerHTML= &#x202a;WPA2 (AES)&#x202c;
```

Comment:
So, once again, I tried `unicode-bidi:isolate` for `mmsTitle` and it did not fix it there. Can I just put it in `<bdi>`?